source/elements/oneTBB/source/flow_graph/join_node_cls.rst

```diff
@@ -119,32 +101,16 @@ Constructs an empty ``join_node`` that belongs to the graph ``g``.
 
 .. code:: cpp
 
-    template<typename B0, typename B1>
-    join_node( graph &g, B0 b0, B1 b1 );
-    template<typename B0, typename B1, typename B2>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2 );
-    template<typename B0, typename B1, , typename B2, typename B3>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B4>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B5>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4, B5 b5 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B5, typename B6>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4, B5 b5, B6 b6 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B5, typename B6, typename B6>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4, B5 b5, B6 b6, B7 b7 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B5, typename B6, typename B7>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4, B5 b5, B6 b6, B7 b7 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B5, typename B6, typename B7, typename B8>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4, B5 b5, B6 b6, B7 b7, B8 b8 );
-    template<typename B0, typename B1, , typename B2, typename B3, typename B5, typename B6, typename B7, typename B8, typename B9>
-    join_node( graph &g, B0 b0, B1 b1, B2 b2, B3 b3, B4 b4, B5 b5, B6 b6, B7 b7, B8 b8, B9 b9 );
+    template <typename B0, typename B1, typename... BN>
+    join_node( graph &g, B0 b0, B1 b1, BN... bn );
```

Comment:
The list of parameters here does not match that in the synopsis. And I think the synopsis is correct, as it should be possible to create a join_node with only one input port.